build.sbt

```diff
@@ -559,6 +575,15 @@ lazy val emberServer = libraryCrossProject("ember-server")
       ProblemFilters.exclude[DirectMissingMethodProblem](
         "org.http4s.ember.server.internal.ServerHelpers.runConnection"
       ),
+      ProblemFilters.exclude[DirectMissingMethodProblem](
+        "org.http4s.ember.server.internal.ServerHelpers.upgradeSocket"
+      ),
+      ProblemFilters.exclude[DirectMissingMethodProblem](
+        "org.http4s.ember.server.internal.ServerHelpers.serverInternal"
+      ),
+      ProblemFilters.exclude[DirectMissingMethodProblem](
+        "org.http4s.ember.server.internal.ServerHelpers.unixSocketServer"
+      ),
```

Comment:
Are these MiMa issues only on Scala 3? If so they should go in the Scala 3 MiMa section below, which we'll remove in the future once MiMa is able to automatically recognize package privates in Scala 3 like it does for Scala 2.

Reply:
Looks like yes.